packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -28,10 +28,10 @@ void main(List<String> args) async {
   Exception? exception;
   final chromedriver = ChromeDriver();
 
-  try {
-    // Start chrome driver before running the flutter integration test.
-    await chromedriver.start();
+  // Start chrome driver before running the flutter integration test.
+  await chromedriver.start();
```

Comment:
why is this moved out of the try block? if an exception is thrown we need to catch it.

Reply:
We rethrow the exception anyway.
But if start fails, finally also fails because late _process in ChromeDriver is not initialized.

Reply:
Let's just make `_process` nullable in the `ChromeDriver` class, and put this back into the try block. We only rethrow on line 62 for exceptions caught in the try block.

Reply:
What is the point to save exception in order to rethrow it?
I would just remove `catch` from here.